docs/duration.md

```diff
@@ -568,21 +568,6 @@ d.total({
 // FIXME: update the result above after duration rounding is fixed
 ```
 
-### duration.**getFields**() : { years: number, months: number, weeks: number, days: number, hours: number, minutes: number, seconds: number, milliseconds: number, microseconds: number, nanoseconds: number }
```

Comment:
Could you drop this commit since it's already in a separate pull request? That would make it easier to review.

Reply:
Oops, I thought Github would handle stacked pull requests. Will rebase.